src/utils/Dimensions.ts

```diff
@@ -0,0 +1,33 @@
+module n3Charts.Utils {
+  'use strict';
+
+  export class Dimensions {
```

Comment:
Also, accessing `Dimension.height` looks like accessing `Axis.verticalAxis`, or... I don't know, `Property.individualProperty`. I mean that's a set, or a collection, right ? Height and width are two dimensions, i.e. one is the normal of the other, in my mind, so... Maybe `DimensionSet` ? :-) 

_we'll find a trade-off ^^_